src/PKSim.Core/Services/ParameterTask.cs

```diff
@@ -301,20 +286,13 @@ public ICommand SetParameterDisplayValueWithoutBuildingBlockChange(IParameter pa
 
       public ICommand SetParameterValue(IParameter parameter, double value, bool shouldUpdateDefaultStateAndValueOriginForDefaultParameter = true)
       {
-         const bool shouldChangeVersion = true;
-
-         if (parameter.IsExpression())
-            return withUpdatedDefaultStateAndValue(commandForRelativeExpressionParameter(parameter, value).Run(_executionContext), parameter, shouldChangeVersion, shouldUpdateDefaultStateAndValueOriginForDefaultParameter);
+         if (parameter.IsExpressionProfile())
```

Comment:
I am not surew about ontogeny factor. Can we change them anywhere? I don't think they are in fact expression profile parameter so I will review this

Reply:
Yes, we can change them in a simulation
![grafik](https://user-images.githubusercontent.com/25061876/142619789-b7eb1c73-549c-4b27-b048-5b801d34d054.png)
Which by the way is kind of inconsistent, if we commit to the individual after that. The individual still shows e.g. "Ontogeny Like CYP3A4". Actually never thought about that :)

Reply:
Yep. But they are not defined in the expression profile. At least they should not since there is no age. Only in the individual

Reply:
true